templates/fine-tune-llm/README.md

```diff
@@ -0,0 +1,82 @@
+# Fine-tuning Llama-2/Mistral models with Anyscale
+
+Anyscale currently offers a simple CLI command to fine-tune LLM models via `anyscale fine-tuning submit`
+and you can take a look at the documentation [here](https://docs.anyscale.com/endpoints/fine-tuning/get-started).
+This guide provides starter configurations if you would like to further customize the fine-tuning process.
+
+### Supported base models
+
+- mistralai/Mistral-7B-Instruct-v0.1
+- meta-llama/Llama-2-7b-hf
+- meta-llama/Llama-2-7b-chat-hf
+- meta-llama/Llama-2-13b-hf
+- meta-llama/Llama-2-13b-chat-hf
+- meta-llama/Llama-2-70b-hf
+- meta-llama/Llama-2-70b-chat-hf
+
+# Step 1 - Launch a fine-tuning job
```

Comment:
can you not run the job on a workspace?

Reply:
Yes you can. Since the jobs may take many hours and the user needs to manually inspect the job log to get the model weight uri, it feels easier to just force the production jobs so that users can always reference their logs. If it's run inside workspaces, the user may lose the job output if run inside the vscode or jupyter